### PR TITLE
Fix preload command support for application IDs

### PR DESCRIPTION
**Pull request by srlowe**
Change-type: patch
Resolves: #2063
Signed-off-by: Scott Lowe <scott@balena.io>

### CODE REVIEW

**Comment by pdcastro** [comment]
We could also amend the description of the `--app` flag, or it could be done as part of #2131.

File: lib/commands/preload.ts [hunk 1]
```diff
@@ -157,6 +157,17 @@ Can be repeated to add multiple certificates.\
 			);
 		}
 
+		// balena-preload currently does not work with numerical app IDs
+		// Load app here, and use app slug from hereon
```
[the remark this answers is not in the document]
Comment (srlowe) [reply lib/commands/preload.ts:161]: Haha, well if we plan to drop numerical IDs, then there's no point in spending long on it :-)

**Comment by srlowe** [comment]
@pdcastro Yeah I'll amend the --app message in 2131.  I've updated that error message here though.

**Review by pdcastro — APPROVED** [review]

**Comment by pdcastro** [comment]
@balena-ci retest

**Comment by pdcastro** [comment]
@balena-ci rebase

**Pull request merged**